README.md

```diff
@@ -32,6 +33,33 @@ Read the [Changelog](https://github.com/Microsoft/vscode-go/blob/master/CHANGELO
 - [Code of Conduct](#code-of-conduct)
 - [License](#license)
 
+## We are moving!
+
+🏡 🚚 🏡 
+
+Our new home will be https://github.com/golang/vscode-go.
+
+As the extension grows in popularity and supports more features, it has required more maintenance and support.
+Ever since the introduction of Go modules, we have been collaborating with the Go team for module support.
+In 2019, we added support for `gopls`, the [Go language server](https://golang.org/s/gopls/README.md) from the
+Go team which has resulted in a closer collaboration. More exciting work is on the horizon with the
+[Go debug adapter service](https://github.com/go-delve/delve/tree/master/service/dap) in delve which will help
+in having a better debugging experience altogether.
+
+Moving to the [GitHub org for golang](https://github.com/golang) brings us closer to the home of other tools
+and libraries the Go extension depends on. We hope that this will help us remain abreast of relevant language
```

Comment:
A simplification:

We hope this will help keep the extension tightly coupled with the Go language and encourage more Gophers to get involved.

Reply:
Fixed in https://github.com/microsoft/vscode-go/pull/3237/commits/b9071ad8d7037db5b8c33790ea969580de42f3d1